curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md

```diff
@@ -11,6 +11,14 @@ In the previous step you turned the words `link to cat pictures` into a link by
 
 In the text of your `p` element, turn the words `cat photos` into a link to `https://freecatphotoapp.com` by adding opening and closing anchor (`a`) tags around these words.
```

Comment:
You did add in the new instructions like the ticket mentioned.
However, you also kept the old instructions too so now the entire group of instructions doesn't make sense

It currently read like this:

```

In the previous step you turned the words link to cat pictures into a link by placing them between opening and closing anchor (a) tags. You can do the same to words inside of an element, such as a p element.

In the text of your p element, turn the words cat photos into a link to https://freecatphotoapp.com by adding opening and closing anchor (a) tags around these words.

You can turn any text into a link, such as the text inside of a p element.

<p>I think <a href="https://www.freecodecamp.org/">freeCodeCamp</a> is great.</p>
In the text of your p element, turn the words cat photos into a link by adding opening and closing anchor (a) tags around these words. Then set the href attribute to https://freecatphotoapp.com/
```

You will need to fix this so we only see the new instructions and there are no duplicate sentences

Reply:
Whoops my bad! I think I fixed the new instructions now. 
I'm not sure why some of the checks are no longer passing though...  :(

Reply:
I am not sure why the e2e tests are failing.
maybe one of the other reviewers can spot why.
I tried looking into the logs but couldn't find any helpful error messages